packages/client/src/components/PlaceholderPages/ContactUs/ContactUs.component.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import './ContactUs.styles.css';
+
+export const ContactUs = () => {
+  return (
+    <main className="contactUs-wrapper">
+      <img
+        className="contactUs-img"
+        src="/assets/images/contactUs.png"
+        alt="contactUs"
```

Comment:
it would be nice if the alt attribute could be more descriptive about the image.